src/components/CodeView/index.tsx

```diff
@@ -31,9 +31,16 @@ const SLOW_LOADING_LINE_COUNT = 1000;
 // This function mimics what https://github.com/rexxars/react-refractor does,
 // but we need a different layout to inline comments so we cannot use this
 // component.
-const renderHighlightedCode = (code: string, language: string) => {
-  const ast = refractor.highlight(code, language);
-  const value = ast.length === 0 ? code : ast.map(mapWithDepth(0));
+const renderCode = (
```

Comment:
Maybe we should update the comment above to mention that, in addition to highlighting code, it can also render non-highlighted code. The name of the function makes it clear but the comment above needs a new sentence I think (we should keep the rest of the comment because it is important).